### DISCUSSION TITLE
Empty "initialTitle"

### DISCUSSION THREAD

**Discussion by Ly-sec** [ghostty-org/ghostty#3606] [Feature Requests, Ideas] [answered]
Hello,
I was just trying to put ghostty in the ignore list in my waybar (`wlr/taskbar`) and since that didn't work I tried getting the title via `hyprctl clients`.

That's the current output:
```
Window 5c7d011b16d0 -> ~/dotfiles:
	mapped: 1
	hidden: 0
	at: 2588,77
	size: 835,1346
	workspace: 4 (4)
	floating: 0
	pseudo: 0
	monitor: 0
	class: com.mitchellh.ghostty
	title: ~/dotfiles
	initialClass: com.mitchellh.ghostty
	initialTitle: 
	pid: 412961
	xwayland: 0
	pinned: 0
	fullscreen: 0
	fullscreenClient: 0
	grouped: 0
	tags: 
	swallowing: 0
	focusHistoryID: 1
	inhibitingIdle: 0
```

It would be nice if `initialTitle` could be set so we can actually work with it. I did try using the class instead but that didn't work.
At least I assume that this would be the issue if I compare the output from `kitty` it seems like that's why it works with kitty.

**Comment by mitchellh** [comment]
cc @jcollie Do you know how to do this?

**Reply by Ly-sec** [reply]
If it helps, kitty just passed their class (I assume)

**Comment by pluiedev** [comment, marked as answer]
I think this is #5934
